source/chapters/user_interface.rst

```diff
@@ -458,6 +458,18 @@ control relative mode:
   to that cue point.
 * **Hot**: The track will seek to the nearest previous :term:`hotcue` point.
 
+**Vinyl button**
```

Comment:
LGTM
Vinyl & Pass appear first in the image so I suggest to move those sections up above `Abs/Rel/Const button` so they're also described first.

Reply:
Done.